drizztsaga/tpa/ds-core.tpa

```diff
@@ -2016,6 +2017,8 @@ COPY ~DRIZZTSAGA/ITM/F_RAISER.ITM~           ~override~
   SAY UNIDENTIFIED_DESC @307 
   SAY DESC @308
 
+COPY ~DRIZZTSAGA/ITM/F_REGAMU.ITM~          ~override~       
```

Comment:
this item already exist on BG2 so no need of name and description... but not on BG1; my mistake. 

So it will create new lines to translate for BGEE for maybe nothing (only for lore, because it is close to useless in DS)

*what do you think, do I removed it or do I create those new lines to translate ?*

Reply:
yes, I will but could you provide tra in french, italian, polish, russian and spanish ? I don't have dialog.tlk for those. Or could you send me those dialog.tlk ?

Reply:
Yeah, I can take it from BG2EE as it provides multiple lang pack.

Reply:
// BG2 #55781
@515 = ~Ruby Pendant~
// BG2 #2309
@516 = ~Ruby Pendant of Beguiling

A favorite toy of the halfling thief Regis, perfect for making people see his point of view.

STATISTICS

Dire Charm abilities once per day.

Not Usable By:
 Wizard Slayer~